<script>
  // No try/catch
  // - await가 던진 에러를 잡을 수 없음
  // - Promise가 거부 상태(Rejected)가 되어버림
  async function f() {
    let res = await fetch('nomeaningurl', { mode: "no-cors" });
    alert(res);
  }

  f(); // uncaught (in promise) TypeError: Failed to fetch  // Rejected Promise
</script>